compression/lempel_ziv_decompress.py

```diff
@@ -0,0 +1,110 @@
+"""
+    One of the several implementations of Lempel–Ziv–Welch decompression algorithm
+    https://en.wikipedia.org/wiki/Lempel%E2%80%93Ziv%E2%80%93Welch
+"""
+
+import sys
+import math
+
+
+def read_file_binary(file):
```

Comment:
```suggestion
def read_file_binary(file_path: str) -> bytes:
```
Type hints? Doctests? See CONTRIBUTING.md.